src/components/simple-billboard/index.js

```diff
@@ -60,11 +60,11 @@ const SimpleBillboard = ({ metric, statusTrend = {}, title }) => {
       difference > 0
         ? {
             type: 'uparrow',
-            fill: '#02865B',
+            fill: '#9EA5A9',
```

Comment:
If the `fill` will be the same, isn't it best to just update [this line](https://github.com/newrelic/nr-labs-components/blob/28b2d7eeb81d6568ab136409612fed2baa27be4e/src/components/simple-billboard/index.js#L74) to be static.